clients/cobol-lsp-vscode-extension/patches/iconv-lite+0.6.3.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/node_modules/iconv-lite/encodings/sbcs-data.js b/node_modules/iconv-lite/encodings/sbcs-data.js
```

Comment:
Oh my... I hope that you know what are you doing =)

Reply:
> Can you, please, add a unit test with encoding? It will be better to add two: one with existing `iconv-lite` codepage and one with patched.

Updated PR